cluster-autoscaler/main.go

```diff
@@ -445,3 +452,37 @@ func validateMinMaxFlag(min, max int64) error {
 func minMaxFlagString(min, max int64) string {
 	return fmt.Sprintf("%v:%v", min, max)
 }
+
+func parseMultipleGpuLimits(flags MultiStringFlag) ([]context.GpuLimits, error) {
```

Comment:
Ideally we should have some unittest for parsing. I can accept adding it later if you create an issue for it.

Reply:
Good catch. I have - some. Just forgot to backport it from previous PR version.

Reply:
Hmm - actually I must have dreamt something. I see no tests. Will add some.